idea-plugin/src/main/kotlin/arrow/meta/ide/plugins/union/UnionIdePlugin.kt

```diff
@@ -0,0 +1,14 @@
+package arrow.meta.ide.plugins.union
+
+import arrow.meta.Plugin
+import arrow.meta.ide.IdeMetaPlugin
+import arrow.meta.invoke
+import arrow.meta.plugins.union.suppressTypeMismatchOnNullableReceivers
+import org.jetbrains.kotlin.diagnostics.Diagnostic
+
+val IdeMetaPlugin.uniontypes: Plugin
+  get() = "Union Types"{
```

Comment:
```suggestion
  get() = "Union Types" {
```

Reply:
I didnt see the comment up until now. I will add it in the next pr